### ISSUE TITLE
Postgresql 'ERROR: invalid name syntax'

### ISSUE THREAD

**Issue by swaxolez** [willbryant/kitchen_sync#61]
I'm getting the following error when I try and make a connection:

Kitchen Sync
Error in the 'from' worker: ERROR:  invalid name syntax

SELECT tablename FROM pg_tables WHERE schemaname = ANY (current_schemas(false)) ORDER BY pg_relation_size(tablename::text) DESC, tablename ASC
Connection closed
Kitchen Syncing failed.

My command is: 

ks --from "postgresql://somebody:password@localhost:5432/testdb"  --to "mysql://somebody:password@10.0.1.39/test_rep" --only "contacts"

**Comment by willbryant** [comment]
I haven't seen that error message before and I'm not sure what it means.  The only really solid hit on google is to do with a user having tables with non-ASCII characters in the name and an unusual collation, and a postgresql patch was implemented to fix it.

What version of postgresql are you using? Do you have any tables with non-ASCII characters in their name in the `testdb` schema?

**Comment by willbryant** [comment]
Also please paste the output of the `locale` command (shell, not postgresql).

**Comment by tomSDL** [comment]
getting a similar error

Error in the 'from' worker: ERROR:  relation "table1" does not exist

SELECT tablename FROM pg_tables WHERE schemaname = ANY (current_schemas(false)) ORDER BY pg_relation_size(tablename::text) DESC, tablename ASC

**Comment by willbryant** [comment]
I haven't been able to find a way to reproduce this myself, even trying out having tables with non-ASCII characters in their names.

@tomSDL What version of postgresql are you using? Could you please paste the output of the locale command (shell, not postgresql)?

**Comment by willbryant** [comment]
Wait, @tomSDL your error is unrelated.  Please open a separate ticket and paste in your command line.

**Comment by swaxolez** [comment]
It's working.  I had some tables with upper case names and quotes around field names.  Many years ago I uploaded some table data from a MS Access db.  That was the culprit.

**Comment by willbryant** [comment]
Ah interesting!  Are you able to dump the problematic bit of schema here @swaxolez?

I did try to support field names including weird characters so I'd like to try and create a test case and fix it, or at least add it to the documentation if I can't easily fix it.

**Comment by willbryant** [comment]
Found it:

```
$ createdb issue61
$ psql issue61
psql (11.2)
Type "help" for help.

issue61=# create table """quotes""_in_the_name" (a int);
CREATE TABLE
issue61=# SELECT tablename FROM pg_tables WHERE schemaname = ANY (current_schemas(false)) ORDER BY pg_relation_size(tablename::text) DESC, tablename ASC;
ERROR:  invalid name syntax
```

**Comment by willbryant** [comment]
Fixed released in v1.16.
